notebooks/6_calculate_statistics.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<p>I didn't quite follow as to why you created <code>titanic_subset</code> and did a groupby on it, and then separately did a groupby on <code>titanic</code>, even though they have the same result (except for the data type of the result). I think there is a teaching point you were trying to emphasize by making that distinction, but I couldn't quite figure it out!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Thanks for noticing, I'll try to improve the _storyline_ . The reason to have the subset is to overcome the necessity of selecting a column (e.g. <code>Age</code>) after the <code>groupby</code> operator. As such, I can first explain </p><p><code>titanic.groupby("Sex").mean()</code>  before doing <code>titanic.groupby("Sex")["Age"].mean()</code>. </p><p> </p><p>Maybe rather work without subset, just do <code>titanic.groupby("Sex").mean()</code>, get a grouped-average result on all the columns (PassengerId,Survived,Pclass,Age,SibSp,Parch and Fare) and next, explain how to just go for a single column of interest? </p>

Reply:
<p>After trying out, I rather stick to:</p><p><code>titanic[["Sex", "Age"]].groupby("Sex").mean()</code> as the solution of the specific question. It is putting less attention to the subset as it was before, but still keep the focus on those two columns involved.</p>